### DISCUSSION TITLE
Pros and cons of updating Antd from 3.26.19 to 4.19.3

### DISCUSSION THREAD

**Discussion by sarahdialmeida** [ant-design/ant-design#34639] [General]
Hi everyone!

I'm working on a project with a very old Antd version (3.26.19), but me and my team are considerating upgrade to the new version and we would like to study the pros and cons first to understand the costs.

**Comment by yoyo837** [comment]
The biggest changes is `Icon` and `Form`/`Form.Item`. https://ant.design/docs/react/migration-v4.
